graphql/pupil/mutations.ts

```diff
@@ -361,4 +363,43 @@ export class MutatePupilResolver {
         await invalidatePupilScreening(pupilScreeningId);
         return true;
     }
+
+    @Mutation(() => Boolean)
+    @Authorized(Role.ADMIN)
+    async adminIncreasePupilGrades() {
+        const pupils = await prisma.pupil.findMany({
+            where: {
+                grade: { not: null, notIn: ['13. Klasse', '14. Klasse'] },
+                OR: [{ gradeUpdatedAt: { lt: moment().subtract(10, 'months').toDate() } }, { gradeUpdatedAt: { equals: null } }],
+            },
+        });
+        logger.info(`Attempting to increase the grade of ${pupils.length} pupils`);
+        let validCount = 0;
+        const batchSize = 1000;
```

Comment:
I tested this with 20k records and it took ~20 seconds, so I guess that's fine for a one-time execution per year?